tools/packaging/static-build/cache_components.sh

```diff
@@ -103,8 +113,14 @@ cache_rootfs_artifacts() {
 			root_hash_vanilla=""
 			root_hash_tdx="${repo_root_dir}/tools/osbuilder/root_hash_tdx.txt"
 		fi
+		if [ "${TEE}" == "sev" ]; then
+			rootfs_tarball_name="kata-static-cc-sev-rootfs-initrd.tar.xz"
+			aa_kbc="online_sev_kbc"
+			image_type="initrd"
+			initramfs_version="-$(get_initramfs_image_name)"
+		fi
 	fi
-	local current_rootfs_version="${osbuilder_last_commit}-${guest_image_last_commit}-${agent_last_commit}-${libs_last_commit}-${attestation_agent_version}-${gperf_version}-${libseccomp_version}-${pause_version}-${rust_version}-${image_type}-${aa_kbc}"
+	local current_rootfs_version="${osbuilder_last_commit}-${guest_image_last_commit}-${agent_last_commit}-${libs_last_commit}-${attestation_agent_version}-${gperf_version}-${libseccomp_version}-${pause_version}-${rust_version}-${image_type}-${aa_kbc}${initramfs_version}"
```

Comment:
Same comment here about the positioning of the args.

Reply:
Changed the order.